src/hashing.rs

```diff
@@ -287,31 +290,35 @@ impl DomainSeparation for MacDomain {
 /// ```
 ///
 /// ```
-/// use sha3::Sha3_256;
-/// use tari_crypto::hashing::Mac;
-/// use tari_utilities::hex::to_hex;
+/// # use sha3::Sha3_256;
+/// # use tari_crypto::hashing::{Mac, MacDomain, DomainSeparation};
+/// # use tari_utilities::hex::to_hex;
```

Comment:
I usually leave these in, especially the first example on a page, because it helps to know what you're bringing into the namespace when reading examples.